src/main/java/bob/Storage.java

```diff
@@ -0,0 +1,142 @@
+package bob;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents a storage that deals with loading tasks from the file and saving tasks in the file
+ */
+public class Storage {
+
+    String filePath;
+    //Parser parser;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Loads file content and re-initialises task if file exists, else creates new file
+     *
+     * @return ArrayList</Task> the list of tasks that were recorded in the existing file, else an empty list
+     * @throws BobException
+     */
+    public ArrayList<Task> load() throws BobException {
```

Comment:
Should some parts in this method be extracted out? As per [code quality guidelines](https://nus-cs2103-ay2324s1.github.io/website/se-book-adapted/chapters/codeQuality.html), we should avoid long methods.